### ISSUE TITLE
The Theme editor should show the parser error when an invalid color is used

### ISSUE THREAD

**Issue by ericpgreen2** [rilldata/rill#4675]
I was confused when my dashboard preview would bounce back to the dashboard editor. I only discovered that the theme color I picked was invalid when I checked my terminal and found the following warning:

![image](https://github.com/rilldata/rill/assets/14206386/c2a1ab3b-e015-4abf-9b05-bb157f3941b2)

**Comment by mindspank** [comment]
@ericpgreen2 if it will be solved by the common editor it feels like we can defer this until the common editor lands?

**Comment by ericpgreen2** [comment]
> @ericpgreen2 if it will be solved by the common editor it feels like we can defer this until the common editor lands?

Agree, this one isn't critical, just further motivation to refactor to a common Editor.
